CefSharp.OutOfProcess.Interface/IOutOfProcessClientRpc.cs

```diff
@@ -35,8 +36,22 @@ public interface IOutOfProcessClientRpc
         /// <param name="parentHwnd">parent Hwnd</param>
         /// <param name="url">start url</param>
         /// <param name="browserId">browser id</param>
+        /// <param name="requestContextPreferences">Request Context Preferences to set.</param>
         /// <returns>Task</returns>
-        Task CreateBrowser(IntPtr parentHwnd, string url, int browserId);
+        Task CreateBrowser(IntPtr parentHwnd, string url, int browserId, IDictionary<string, object> requestContextPreferences);
+
+        /// <summary>
+        /// Modify RequestContext-Preferences for an existing browser.
+        /// </summary>
+        /// <param name="browserId">browser id</param>
+        /// <param name="requestContextPreferences">Request Context Preferences to set.</param>
+        void UpdateRequestContextPreferences(int browserId, IDictionary<string, object> requestContextPreferences);
```

Comment:
Please rename method to `SetRequestContextPreferences` so it matches `RequestContext.SetPreference` naming.

Reply:
done